src/components/Slider/Slider.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import { SliderContainer, SliderInput, InputValue } from './styles';
+
+
+const Slider = () => {
+  const [value, setValue] = useState(0);
+
+  const handleChange = (e) => {
+    setValue(e.target.value);
+  };
+
+  return (
+    <SliderContainer>
+      <div><InputValue>{value}</InputValue></div>
+      <div>
+        <SliderInput
+          type="range"
+          min="1"
+          max="10"
```

Comment:
move this attributes in constants please

Reply:
do you mean to put in constants in this file as ``const min = 1;```? or in file ```constants.js```?

Reply:
I mean this values should comes from `props` (except of `type="range"`)